api/clever/__init__.py

```diff
@@ -154,6 +162,13 @@ def oauth_callback(self, _db, params):
         )
         patron._external_type = external_type
 
+        credential, is_new = get_one_or_create(
+            _db, Credential, data_source=self._data_source(_db),
+            type=self.TOKEN_TYPE, patron=patron,
+        )
+        credential.credential = token
```

Comment:
This is the token that clever gives us. We never use it again, so we could also generate our own token and store that instead.